fuel-core/src/args.rs

```diff
@@ -12,14 +19,17 @@ lazy_static::lazy_static! {
 pub const LOG_FILTER: &str = "RUST_LOG";
 pub const HUMAN_LOGGING: &str = "HUMAN_LOGGING";
 
-#[derive(Parser, Debug)]
+#[derive(Parser, Clone, Debug)]
 #[clap(
     name = "fuel-core",
     about = "Fuel client implementation",
     version,
     rename_all = "kebab-case"
 )]
 pub struct Opt {
```

Comment:
https://github.com/FuelLabs/fuel-core/issues/457

Reply:
I was wondering if it was worth re-vamping the cli structure with this PR but wanted to keep changes minimal. I definitely agree though because this implementation is a bit clunky